whylog/assistant/regex_assistant/regex.py

```diff
@@ -0,0 +1,116 @@
+"""
+Regex verification and creating (but not finding groups in regex)
+"""
+import re
```

Comment:
you use `re` here, but @kgromadzki uses `regex` if it can be loaded. In some cases this will cause verification of regex to fail (for example if there are more than 100 groups). Please find a way to import it consistently without duplicating code.